webapp/pages/terms-and-conditions.vue

```diff
@@ -19,5 +77,13 @@ export default {
       title: this.$t('site.termsAc'),
     }
   },
+  data() {
+    return {
+      confirmed: false,
+      latestDate_TaC: '17.10.2008',
+      latestDate_TaC_update: '23.10.2008',
```

Comment:
```suggestion
      termsAndConditionsVersion: '1.0.0',
```
@ogerly :
* Use camelCase
* Use JS Date objects if possible
* In this case I would suggest to use a version